Angular/src/app/app.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class Service {
+    getEmployees() {
+        let employees = [{
```

Comment:
Now a new array of employees will be created on each getEmployees method call. Let's define employees at the top as const and make an interface for an Employee type to better match Angular patters